localization/ch.xml

```diff
@@ -1,57 +1,57 @@
-    <state name="Zug (de)" iso_code="CH-ZG" country="CH" zone="Europe" tax_behavior="0" />
-    <state name="Zürich (de)" iso_code="CH-ZH" country="CH" zone="Europe" tax_behavior="0" />
+    <state name="Aargau (de)" iso_code="CH-AG" country="CH" zone="Europe (non-EU)" tax_behavior="0" />
+    <state name="Appenzell Ausserrhoden (de)" iso_code="CH-AR" country="CH" zone="Europe (non-EU)" tax_behavior="0" />
+    <state name="Appenzell Innerrhoden (de)" iso_code="CH-AI" country="CH" zone="Europe (non-EU)" tax_behavior="0" />
+    <state name="Basel-Landschaft (de)" iso_code="CH-BL" country="CH" zone="Europe (non-EU)" tax_behavior="0" />
+    <state name="Basel-Stadt (de)" iso_code="CH-BS" country="CH" zone="Europe (non-EU)" tax_behavior="0" />
+    <state name="Bern (de), Berne (fr)" iso_code="CH-BE" country="CH" zone="Europe (non-EU)" tax_behavior="0" />
+    <state name="Fribourg (fr), Freiburg (de)" iso_code="CH-FR" country="CH" zone="Europe (non-EU)" tax_behavior="0" />
+    <state name="Genève (fr)" iso_code="CH-GE" country="CH" zone="Europe (non-EU)" tax_behavior="0" />
+    <state name="Glarus (de)" iso_code="CH-GL" country="CH" zone="Europe (non-EU)" tax_behavior="0" />
+    <state name="Graubünden (de)" iso_code="CH-GR" country="CH" zone="Europe (non-EU)" tax_behavior="0" />
+    <state name="Jura (fr)" iso_code="CH-JU" country="CH" zone="Europe (non-EU)" tax_behavior="0" />
+    <state name="Luzern (de)" iso_code="CH-LU" country="CH" zone="Europe (non-EU)" tax_behavior="0" />
+    <state name="Neuchâtel (fr)" iso_code="CH-NE" country="CH" zone="Europe (non-EU)" tax_behavior="0" />
+    <state name="Nidwalden (de)" iso_code="CH-NW" country="CH" zone="Europe (non-EU)" tax_behavior="0" />
+    <state name="Obwalden (de)" iso_code="CH-OW" country="CH" zone="Europe (non-EU)" tax_behavior="0" />
+    <state name="Sankt Gallen (de)" iso_code="CH-SG" country="CH" zone="Europe (non-EU)" tax_behavior="0" />
+    <state name="Schaffhausen (de)" iso_code="CH-SH" country="CH" zone="Europe (non-EU)" tax_behavior="0" />
+    <state name="Schwyz (de)" iso_code="CH-SZ" country="CH" zone="Europe (non-EU)" tax_behavior="0" />
+    <state name="Solothurn (de)" iso_code="CH-SO" country="CH" zone="Europe (non-EU)" tax_behavior="0" />
+    <state name="Thurgau (de)" iso_code="CH-TG" country="CH" zone="Europe (non-EU)" tax_behavior="0" />
+    <state name="Ticino (it)" iso_code="CH-TI" country="CH" zone="Europe (non-EU)" tax_behavior="0" />
+    <state name="Uri (de)" iso_code="CH-UR" country="CH" zone="Europe (non-EU)" tax_behavior="0" />
+    <state name="Valais (fr), Wallis (de)" iso_code="CH-VS" country="CH" zone="Europe (non-EU)" tax_behavior="0" />
+    <state name="Vaud (fr)" iso_code="CH-VD" country="CH" zone="Europe (non-EU)" tax_behavior="0" />
+    <state name="Zug (de)" iso_code="CH-ZG" country="CH" zone="Europe (non-EU)" tax_behavior="0" />
+    <state name="Zürich (de)" iso_code="CH-ZH" country="CH" zone="Europe (non-EU)" tax_behavior="0" />
```

Comment:
Nice catch! I guess the ID should be turned into `Europe_out_E_U` instead and also be notified in the [install-dev/data/xml/country.xml file](https://github.com/PrestaShop/PrestaShop/blob/develop/install-dev/data/xml/country.xml) by changing the `contains_states` value to `1`.